README.md

```diff
@@ -67,3 +67,40 @@ If you are using a non-standard IP address or port for your Scylla instance, you
 ```bash
 SCYLLA_URI=172.16.0.1:9042 cargo test -- --test-threads=1
 ```
+
+#### cassandra-stress frontend python tests
+There are multiple python scripts testing cassandra-stress frontend in `tools` directory.
+
+These scripts require scylla python driver, which can be installed using `pip`:
+```bash
+pip install scylla-driver
+```
+
+These tests require running healthy scylla Docker container.
+You can provide own docker container's ip with `--scylla-ip` argument, or
+let the scripts set up the container for you.
+
+Example usage:
+```bash
+# test_cs_equal_db.py test runs both cassandra-stress and cql-stress-cassandra-stress
+# write routines, and checks whether resulting tables are equal.
+# You can specify the number of inserted rows with --workload-size argument.
+
+# To understand the specifics of other tests, check the comments in the corresponding scripts.
+
+# Run the test with your own docker container.
+docker run --name scylla-ci -d -p 9042:9042 scylladb/scylla # OR `docker compose -f docker/scylla_test/compose.yml up -d --wait`
+./tools/test_cs_equal_db.py --scylla-ip 127.0.0.1 --workload-size 1M
+
+# Or let the script to set up the container for you.
+./tools/test_cs_equal_db.py --workload-size 1M
+```
+
+To run the test cases used during CI (defined in `./tools/cassandra_stress_ci.py`), you can make use of [pytest](https://pytest.org).
+Before running the tests, make sure you have scylla up and running.
+If you use some non-standard scylla URI, you can specify it via `SCYLLA_URI` env variable.
+```bash
+docker compose -f docker/scylla_test/compose.yml up -d --wait
+export SCYLLA_URI="127.0.0.1:9042"
+pytest -s ./tools/cassandra_stress_ci.py
+```
```

Comment:
The docs seem outdated. For example, there is no `test_cs_equal_db.py` script anymore.